packages/harmony/src/foundations/typography/typography.ts

```diff
@@ -36,16 +36,16 @@ export const typography = {
     '9xl': 96
   },
   lineHeight: {
-    xs: 12,
-    s: 16,
-    m: 20,
-    l: 24,
-    xl: 32,
-    '2xl': 40,
-    '3xl': 52,
-    '4xl': 68,
-    '5xl': 88,
-    '6xl': 120
+    xs: '12px',
+    s: '16px',
+    m: '20px',
+    l: '24px',
+    xl: '32px',
+    '2xl': '40px',
+    '3xl': '52px',
+    '4xl': '68px',
+    '5xl': '88px',
+    '6xl': '120px'
```

Comment:
Yeaaaa, I ran into this too and am 50/50 on this. I like everything being simple ints and not string bc the ints are more flexible in terms of usage, but I do recognize that it is annoying to have to add the px manually for line heights.

Reply:
To me pretty annoying to interp the px, and since the number is actually valid I feel like folks might get confused by it, so I'm leaning towards using explicit string